hack/extra-values.yaml

```diff
@@ -0,0 +1,24 @@
+auth:
+  # The name of the secret to look for the config in. Or, the name of the
+  # secret to be created and for the credentials to be provided to if a secret
+  # does not yet exist and createSecret is not overriden to false.
+  secretName: azure-creds
+  # Allows the plugin to automatically create the secret containing the
+  # credentials if it does not yet exist.
+  createSecret: true
+  # The tenant ID for the application in Microsoft Entra.
+  tenantId: ""
+  # The client ID associated with the tenant. Also known as application ID.
+  clientId: ""
+  # The client secret associated with the tenant.
+  clientSecret: ""
+# The subscription ID for a subscription.
```

Comment:
L15-24 can be removed, right?

Reply:
ty, fixed

Keeping this file around for reference even though we're not working with Helm yet because it's used to know what must be created for a secret for DevSpace to use.